packages/components/dropdown/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@spark-ui/dropdown",
-  "version": "0.21.5",
+  "version": "1.0.0",
```

Comment:
I think that if you do it like this, you will publish `1.1.0`. Perhaps you should do `1.0.0-0`

Reply:
See [example](https://github.com/adevinta/spark/pull/1659/files#diff-b023b4d5257c64b23cb36f5d36594f7e6c0d80deca00ee635f5a69a86163e187R3)

Reply:
It would publish 1.0.1 not 1.1.0 because I declared it as a fix but you are right :)

I didn't know this trick.